road/scatter.py

```diff
@@ -0,0 +1,236 @@
+import os
+import json
+import yaml
+import pandas as pd
+from sklearn.linear_model import LinearRegression
+
+
+def compute_and_save_errors(
+        est_df,
+        obs_df,
+        chosen_timeperiod,
+        combined_df_cols,
+        classification_col):
+    # Compute errors
+    times = ['Daily', 'AM', 'MD', 'PM', 'EV', 'EA']
+    error = {col: est_df[col] - obs_df[col] for col in times}
+    error_df = pd.DataFrame(error)
+    error_squared_df = error_df.pow(2)
+    error_percent_df = error_df.divide(obs_df[times]).fillna(0)
+    error_percent_df = error_percent_df.apply(lambda x: x * 100, axis=1)
+
+    # Combine dataframes
+    combined_df = pd.concat(
+        [est_df, obs_df[times], error_df, error_squared_df, error_percent_df], axis=1)
+    Select_time_period_df = combined_df.filter(like=chosen_timeperiod)
+    specific_columns_combined_df = combined_df[combined_df_cols]
+    Select_time_period_loc_df = pd.concat(
+        [specific_columns_combined_df, Select_time_period_df], axis=1)

```

Comment:
e.g. for this line, perhaps directly saying
```
[combined_df[combined_df_cols], Select_time_period_df], axis=1)
```
would be clearer, since you only use that variable once and it's pretty clear what that's doing anyways (i.e. the name isn't adding too much---if you feel that it's a little unclear, maybe add a comment, which allows for greater flexibility than just a variable name?)